styles.css

```diff
@@ -0,0 +1,98 @@
+li {
+  list-style: none;
+  display: inline-block;
+}
+
+header {
+  position: absolute;
+  width: 423px;
+  height: 65px;
+  left: 0;
+  top: 0;
+  background: #3c3a39;
+}
+
+.hero {
+  background: #1c1a19;
+  background-image: url("Images/background.png");
+  display: flex;
+  background-position: 100% 100%;
+  flex-direction: column;
+  background-repeat: no-repeat;
+  padding: 20px 24px 5px 24px;
+  position: absolute;
+  width: 375px;
+  height: 590px;
+  left: 0;
+  top: 65px;
+}
```

Comment:
- [ ] [Optional] When adding margins, paddings, widths, and heights to your pages, always favor percent-based units (%, vw, vh) over fixed units (px), except when a fixed unit is really needed. this will help improve the responsiveness of your page.